website/views/static/front.jade

```diff
@@ -75,13 +75,20 @@ html(ng-app='habitrpg', ng-controller='RootCtrl')
         .presslogos.text-center
           = env.t('featuredIn')
           br
-          img(src='https://s3.amazonaws.com/habitrpg-assets/front/images/presslogos/lifehacker.png')
-          img(src='https://s3.amazonaws.com/habitrpg-assets/front/images/presslogos/nyt-logo.png')
-          img(src='https://s3.amazonaws.com/habitrpg-assets/front/images/presslogos/makeuseof.png')
-          img(src='https://s3.amazonaws.com/habitrpg-assets/front/images/presslogos/Forbes_logo.png')
-          img(src='https://s3.amazonaws.com/habitrpg-assets/front/images/presslogos/Cnetlogo.png')
-          img(src='https://s3.amazonaws.com/habitrpg-assets/front/images/presslogos/Fast-Company-logo.png')
-          img(src='https://s3.amazonaws.com/habitrpg-assets/front/images/presslogos/kickstarter-logo.png')
+          a(href='http://lifehacker.com/behind-the-app-the-story-of-habitrpg-1682283507')
```

Comment:
These links should all open in a new tab since they're external, so the `target='_blank'` attribute should be applied to all of them.